pkgs/development/python-modules/debugpy/default.nix

```diff
@@ -49,12 +49,6 @@ buildPythonPackage rec {
     # To avoid this issue, debugpy should be installed using python.withPackages:
     # python.withPackages (ps: with ps; [ debugpy ])
     ./fix-test-pythonpath.patch
-
-    # Fix tests with flask>=2.0
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Oh right, thanks! :slightly_smiling_face: